lib/condor.py

```diff
@@ -179,6 +179,12 @@ def submit(
         if m:
             print(f"{hl}Use job id {m.group(1)}.0@{schedd_name} to retrieve output{hl}")
 
+            # call any job_info commands requested with the jobid
+            for ji in vargs["job_info"]:
```

Comment:
This line makes unit tests fail with a `KeyError`.  We either need to try-except it or have it be something like

```python
job_info = vargs.get("job_info", [])
for ji in job_info
...
```